app/full_text_search.py

```diff
@@ -204,7 +208,7 @@ def search(repo, query_txt, max_results):
 
     results_list = [non_romanized_person_location_index_results,
                     person_location_index_results]
-    index_results = get_person_ids_from_results(query_txt, results_list)
+    index_results = get_person_ids_from_results(query_txt, results_list, returned_fields)
```

Comment:
I think 'record_id' is necessary here as we need to remove duplicates in 'get_person_ids_from_results' according to 'record_id'.